src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -9,18 +9,148 @@
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private Node<K, V>[] nodeArray;
+    private int size;
+    private final static int DEFAULT_CAPACITY = 16;
+    private final static float LOAD_FACTOR = 0.75f;
+    private int threshold;
+    private int newArrayIndex;
```

Comment:
remove this from class level, it is not a class property, you just need it in some separate methods. Also it can cause issues in multithreading execution (Just for common info)

Reply:
@al-pavlenko +